### DISCUSSION TITLE
Entities on screensaver - how to modify?

### DISCUSSION THREAD

**Discussion by JaroslawPrzybyl** [blakadder/nspanel#60] [General]
Here is my screensaver:
![image](https://github.com/blakadder/nspanel/assets/33628620/892b2f7d-f7ba-4a02-a740-5b1074f7ff2c)
Code (part):
 
    screensaver:
      #type: screensaver2    
      #entity: weather.dom_w_lesie
      doubleTapToUnlock: true
      alternativeLayout: true 
      entities:
        - entity: weather.dom_w_lesie
        - entity: delete
        - entity: switch.oswietlenie_zewnetrzne_2
          icon: 
              "on": mdi:gate-open
              "off": mdi:gate        
          name: 'Brama' 
          color:
              "on": [255,0,0]
              "off": [0,255,0] 
          value: 

And my questions, connected to the entity "Brama" on screen, it is a switch:
1) how to disable a text information of state below the icon? The only method as I found is to modify "value" to empty string and it works, but gives small icon and empty space below. Would it be possible to just disable this field and get bigger icon? Or have something similar to first entity: weather: big icon and maybe not so big, but bigger font with name?
2) is there any possibility to have an active button on screensaver, not only presentation of its state?

**Comment by blakadder** [comment]
that has nothing to do with this repo
